Python/Product/EnvironmentsList/PipExtensionProvider.cs

```diff
@@ -72,6 +73,15 @@ public sealed class PipExtensionProvider : IEnvironmentViewExtension {
             _cache = PipPackageCache.GetCache(_index, _indexName);
         }
 
+        [SuppressMessage("Microsoft.Usage", "CA2213:DisposableFieldsShouldBeDisposed", MessageId = "_pipCancel",
```

Comment:
It doesn't look like EnvironmentView ever disposes of it's extensions, so this is never called?